pkg/reconciler/wrappedPod.go

```diff
@@ -67,6 +67,10 @@ func getFlatIPSet(pod v1.Pod) map[string]void {
 
 	for _, network := range networkStatusList {
 		// we're only after multus secondary interfaces
+		if len(network.Interface) == 0 {
```

Comment:
hm, right, never had considered a network without interfaces. Can that happen ?..

This code positioned here renders the above comment invalid; afaiu, all multus secondary interfaces will have the `net` prefix followed by an ordinal.

Reply:
can you add a test in https://github.com/k8snetworkplumbingwg/whereabouts/blob/master/pkg/reconciler/wrappedPod_test.go#L17 that triggers this fix ?

Reply:
In my case, the default pod interface will be displayed in network status annotations without field interface be filled.
like below,
`    k8s.v1.cni.cncf.io/network-status: |-
      [{
          "name": "",
          "ips": [
              "172.16.125.147"
          ],
          "default": true,
          "dns": {}
      },{
          "name": "default/bridge-whereabouts",
          "interface": "net0",
          "ips": [
              "192.168.122.101"
          ],
          "mac": "ea:65:54:2e:a6:e3",
          "dns": {}
...`

Reply:
The multus secondary interfaces can be named by user.

> apiVersion: v1
kind: Pod
metadata:
  name: samplepod
  annotations:
    k8s.v1.cni.cncf.io/networks: bridge-whereabouts@hello,bridge-hostlocal@world
...

Reply:
I mean the interface names on the pods, that will appear listed on the network status, not the networks.

Reply:
the interface `eth0` (not multus secondary interface) with ip `172.16.125.147` will appear listed on the network status like below
`k8s.v1.cni.cncf.io/network-status: |- [{ "name": "", "ips": [ "172.16.125.147" ], "default": true, "dns": {} },....`

Reply:
The reconciler **only** tackles multus secondary interfaces, as per https://github.com/k8snetworkplumbingwg/whereabouts/pull/118#discussion_r670518554. 

Reconciling the primary interface was never a goal.

How about you skip the interface on the network status when it is the `default` network, rather than when its interface does not have a name ? I find that more explicit.

I'm still missing a unit test for this fix, and I wonder if we should not add a log instead of silently ignoring the unnamed interfaces.

Reply:
Right, thanks for your advice! And i'll add a unit test for this fix.

How about just ignoring it? Does the log info here have any help?

Reply:
I'm unsure - which I guess makes your suggestion as good / bad as mine :) 

We can silently ignore it ... It's not an error.